### ISSUE TITLE
connectionstyle curved edges not working

### ISSUE THREAD

**Issue by Jogala** [networkx/networkx#3812]
Hi, 

this code sample should connect the two nodes with curved edges:

```
import networkx as nx

G=nx.MultiDiGraph()

G.add_edge(0,1,weight=1)
G.add_edge(1,0,weight=1)

nx.draw(G,connectionstyle = 'arc3,rad=0.2')
```

But instead it produces: 
 
![Figure_1](https://user-images.githubusercontent.com/5442389/73775434-275fc900-4786-11ea-91ac-e0abcb043240.png)

I am working with

conda list says I am working with: 
networkx                  2.2                      py36_1

**Comment by dschult** [comment]
You should try a more recent version.
This was fixed in #3307 
try ```conda update networkx```

**Comment by Jogala** [comment]
I update matplotlib and networkx, now it is working, thanks.
